WinUIGallery/ControlPages/AnnotatedScrollBarPage.xaml.cs

```diff
@@ -0,0 +1,172 @@
+//*********************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+// THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
+// ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
+// IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
+// PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.
+//
+//*********************************************************
+
+using System.Collections.ObjectModel;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Controls.Primitives;
+using Microsoft.UI.Xaml.Media;
+
+namespace AppUIBasics.ControlPages
+{
+    public sealed partial class AnnotatedScrollBarPage : Page
+    {
+        private const int AzureCount = 32;
+        private const int CrimsonCount = 50;
+        private const int CyanCount = 8;
+        private const int FuchsiaCount = 70;
+        private const int GoldCount = 90;
+        private const int ItemWidth = 120;
+        private const int ItemHeight = 90;
+
+        public ObservableCollection<SolidColorBrush> ColorCollection = new ObservableCollection<SolidColorBrush>();
+
+        public AnnotatedScrollBarPage()
+        {
+            this.InitializeComponent();
+            this.DataContext = this;
+            this.Loaded += AnnotatedScrollBarPage_Loaded;
+
+            PopulateColorCollection();
+        }
+
+        private void AnnotatedScrollBarPage_Loaded(object sender, Microsoft.UI.Xaml.RoutedEventArgs e)

```

Comment:
Will be helpful to add comments explaining methods related to implementing AnnotatedScrollBar.